--- a/meson.build
+++ b/meson.build
@@ -182,6 +182,7 @@
 	dependencies: dependencies,
 	include_directories: inc,
 	install: true,
+	install_rpath: '$ORIGIN/../lib',
 )
 
 executable(
@@ -190,6 +191,7 @@
 	dependencies: ctldependencies,
 	include_directories: inc,
 	install: true,
+	install_rpath: '$ORIGIN/../lib',
 )
 
 scdoc = dependency('scdoc', native: true, required: get_option('man-pages'))
